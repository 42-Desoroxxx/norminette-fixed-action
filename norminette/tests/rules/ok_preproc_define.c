#define TOTO "TATA"
#define TOTO struct->val
#define PRINTF       "printf"
#define TOTO 12
