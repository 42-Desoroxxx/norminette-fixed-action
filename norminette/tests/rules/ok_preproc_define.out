tests/rules/ok_preproc_define.c: OK!
